app/controllers/public_body_controller.rb

```diff
@@ -99,7 +99,7 @@ def view_email
     @public_body = PublicBody.find_by_url_name_with_historic(params[:url_name])
     raise ActiveRecord::RecordNotFound.new("None found") if @public_body.nil?
 
-    I18n.with_locale(I18n.locale.to_s) do
+    I18n.with_locale(AlaveteliLocalization.locale.to_s) do
```

Comment:
I think we should wrap `with_locale` too, as its not super obvious where to use `AlaveteliLocalization` vs `I18n` vs `FastGettext`.